app/services/facilities/website_url_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Facilities
+  class WebsiteUrlService
+    FACILITY_TYPES = {
+      'va_benefits_facility' => 'VBA',
+      'va_health_facility' => 'VHA',
+      'va_cemetery' => 'NCA',
+      'vet_center' => 'VC'
+    }.freeze
+
+    def initialize
+      source = Rails.root.join('lib', 'facilities', 'website_data', 'websites.csv')
+      station_websites = CSV.read(source, headers: true)
+      @websites = map_websites_to_stations(station_websites)
+    end
+
+    def find_for_station(id, type)
+      unique_id = "#{id}_#{FACILITY_TYPES[type]}"
+      entry = @websites[unique_id]
+      entry.nil? ? '' : entry
+    end
+
+    private
+
+    def map_websites_to_stations(station_websites)
+      station_websites.each_with_object({}) do |station, hash|
+        unique_id = "#{station['StationNum']}_#{station['Org']}"
```

Comment:
I don't feel strongly about this, but I wonder if it is worth flipping the order of the fields in `unique_id` so that it matches the id we create in the serializer and send in the response. :man_shrugging:

Reply:
Flipped it and reversed it ◀️